cmd/removebinary_nonwindows.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2015 Matthew Holt and The Caddy Authors
```

Comment:
Could probably rename this file to `removebinary.go`, since it's the "standard" one and the `_nonwindows` suffix doesn't do anything.